homeassistant/components/mqtt/strings.json

```
@@ -7,6 +7,14 @@
     "deprecation_mqtt_legacy_vacuum_discovery": {
       "title": "MQTT vacuum entities with legacy schema added through MQTT discovery",
       "description": "MQTT vacuum entities that use the legacy schema are deprecated, please adjust your devices to use the correct schema and restart Home Assistant to fix this issue."
+    },
+    "entity_name_is_device_name": {
+      "title": "MQTT entities with a name is equal to device name",
+      "description": "Some MQTT entities have an entity name that is equal to the device name. This is not expected. To avoid a duplicate name the entity name is set to `null` as a work-a-round. Please update your configuration, or inform the maintainer of the integration that supplies the entity, and restart Home Assistant to fix this issue.\n\nList of effected entities:\n\n{config}"
```

Small wording improvement, as I think integration is confusing in this context.

```suggestion
      "description": "Some MQTT entities have an entity name equal to the device name. This is not expected. The entity name is set to `null` as a work-a-round to avoid a duplicate name. Please update your configuration, or inform the maintainer of the software application that supplies the entity, and restart Home Assistant to fix this issue.\n\nList of effected entities:\n\n{config}"
```

Additionally (not solved in this suggestion), the sentence has an issue. Informing the upstream author and restarting, doesn't fix the issue... (however, the text implies that).

---

I have splitted manual configured and discovered mqtt items.